### PR TITLE
Skip schema validation by default

### PR DESCRIPTION
**Pull request by yrucrem**

### CODE REVIEW

**Comment by yrucrem** [comment]
runbuild

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -23,6 +23,11 @@ include::content/docs/variables.adoc-include[]
 
 * The support for the embedded Elasticsearch will be dropped in the future. It is highly recommended to link:{{< relref "elasticsearch.asciidoc" >}}#_dedicated_elasticsearch[setup Elasticsearch as a dedicated service].
 
+[[v0.40.1]]
+== 0.40.1 (TBD)
+
+icon:check[] Core: Schema validation will no longer fail by default, when Elasticsearch is not available. The previous behavior can be enforced by setting `SchemaUpdateParameters#setStrictValidation` to `true`.-
```
Comment (Jotschi) [inline CHANGELOG.adoc]:
```suggestion
icon:check[] Core: Schema validation will no longer fail by default, when Elasticsearch is not available. The previous behavior can be enforced by adding `?strictValidation=true`.
```

File: core/src/test/java/com/gentics/mesh/test/context/MeshTestContext.java [hunk 2]
```diff
@@ -500,6 +500,7 @@ public MeshOptions init(MeshTestSetting settings) throws Exception {
 			break;
 		case TRACKING:
 			System.setProperty(TrackingSearchProvider.TEST_PROPERTY_KEY, "true");
+		case BROKEN: // fall-through
```
Comment (Jotschi) [inline core/src/test/java/com/gentics/mesh/test/context/MeshTestContext.java]: Change type to "UNREACHABLE". Add searchOptions.setUrl("http://localhost:1") - Remove the other new search provider code.

File: rest-model/src/main/java/com/gentics/mesh/parameter/SchemaUpdateParameters.java [hunk 3]
```diff
@@ -66,4 +71,25 @@ default SchemaUpdateParameters setBranchNames(String... branchNames) {
...
+		setParameter(STRICT_VALIDATION_KEY, String.valueOf(flag));
+		return this;
+	}
+
+	/**
+	 * Check whether the strict validation flag for search index validation is set.
+	 *
+	 * @return
+	 */
+
```
Comment (Jotschi) [inline rest-model/src/main/java/com/gentics/mesh/parameter/SchemaUpdateParameters.java]: newline

**Pull request merged**